tests/get_targets_from_db_test.py

```diff
@@ -53,16 +53,16 @@ def test_write_target_fasta_files(self, mock_write_fasta_file):
             '17__RPOBgbs__RPOBgbs-2__17',
             '18__RPOBgbs__RPOBgbs-3__18',
             '19__RPOBgbs__RPOBgbs-4__19']
-        write_target_fasta_files(targets, self.TEST_FASTA, 'test_data/CHECK_')
+        write_target_fasta_files(targets, self.TEST_FASTA, 'test_data/output/CHECK_')
         mock_write_fasta_file.assert_has_calls([
-            call(self.TEST_FASTA, '7__PARCGBS__PARCGBS-1__7', 'test_data/CHECK_'),
-            call(self.TEST_FASTA, '5__GYRAGBS__GYRAGBS-1__5', 'test_data/CHECK_'),
-            call(self.TEST_FASTA, '11__23S1__23S1-1__11', 'test_data/CHECK_'),
-            call(self.TEST_FASTA, '12__23S3__23S3-3__12', 'test_data/CHECK_'),
-            call(self.TEST_FASTA, '16__RPOBgbs__RPOBgbs-1__16', 'test_data/CHECK_'),
-            call(self.TEST_FASTA, '17__RPOBgbs__RPOBgbs-2__17', 'test_data/CHECK_'),
-            call(self.TEST_FASTA, '18__RPOBgbs__RPOBgbs-3__18', 'test_data/CHECK_'),
-            call(self.TEST_FASTA, '19__RPOBgbs__RPOBgbs-4__19', 'test_data/CHECK_')
+            call(self.TEST_FASTA, '7__PARCGBS__PARCGBS-1__7', 'test_data/output/CHECK_'),
```

Comment:
Might be better if we put all these test_data/output/ test_data/input directory paths into constants somewhere **at some point**, rather than repeating the same hardcoded file path in all the tests. Easier to change if ever required. e.g a test base class.